### PR TITLE
Update token-swap.hbs

### PR DESCRIPTION
**Pull request by JamesMoc**
Changed FAQs on token-swap page with more up to date info.

Added p tags to make  the page more readable

### CODE REVIEW

File: developer-rchain-coop/src/views/token-swap.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="masthead">
     <div id="logoWrapper">
-        <img src="/assets/rchain-redwhite-logo.png" alt="rchain">
+        <img src="https://developer.rchain.coop/assets/rchain-redwhite-logo.png" alt="rchain">
```
Comment (dckc) [inline developer-rchain-coop/src/views/token-swap.hbs:3]: this change looks spurious; was it on purpose?

File: developer-rchain-coop/src/views/token-swap.hbs [hunk 2]
```diff
@@ -16,49 +16,82 @@
...
+                        <li class="body"> Your RHOC was not in a multisig wallet or smart contract.</li>
+                    </ul>
+                    <p>If both the above are the case then do the following.</</p>
+                </div>
+                <div class="title">How can I see my REV balance?</div>
+                <div class="body">
+                    <p>There will be a few methods.' Here are two:</p>
+                </div>
+                    <ul>
+                        <li class ="body">View with <a class="inline-link" target="_blank" href="https://revdefine.io/#/explorer/rev">Revdefine</a> block explorer</li>
```
Comment (dckc) [inline developer-rchain-coop/src/views/token-swap.hbs:55]: Linking directly to revdefine.io carries a burden to watch it dilligently to be sure it continues to behave in the best interest  of the community. I don't see time in my schedule to do that. Does anyone else?

File: developer-rchain-coop/src/views/token-swap.hbs [hunk 3]
```diff
@@ -16,49 +16,82 @@
...
+                        <li class ="body">View with <a class="inline-link" target="_blank" href="https://myrchainwallet.com">MyRchainWallet</a></li>
```
Comment (dckc) [inline developer-rchain-coop/src/views/token-swap.hbs:56]: linking directly to a wallet is even riskier; it encourages people to put their private keys into that software. I'm inclined to link to a discussion forum such as #rhoc2rev and encourage people to ask about block explorers and wallets there.

We could perhaps use github issues for public forum style discussion of FAQs that are changing. Or stackoverflow or reddit.

File: developer-rchain-coop/src/views/token-swap.hbs [hunk 4]
```diff
@@ -16,49 +16,82 @@
...
-                <div class="title">My RHOC is stored in a hardware wallet, is this supported at launch?</div>
-                <div class="body">As long as you control the private key, this conforms to the criteria mentioned above.</div>
+                <div class="body">
+                    <p>No. Multi-signature wallets are not supported. The issuance of REV requires you to control your Ethereum private keys to have received your REV. To receive REV for RHOC you must have had your RHOC in an Ethereum account where you control the private keys before the finalization of RHOC balances, which occured on Jan 28, 2020, just prior to the launch of the RChain blockchain on Feb 25, 2020.</p>
+                </div>
+                <div class="title">My RHOC was stored in an exchange, is this supported at the launch?</div>
+                <div class="body">
+                    <p>If the exchange was Kucoin and you didn't withdraw your RHOC prior to May 8 2020 Kucoin will have seized your REV.</p>
```
Comment (dckc) [inline developer-rchain-coop/src/views/token-swap.hbs:84]: Kucoin's position is not something we want to take on as our own. "KuCoin claims X" is as far as I would go.

**Comment by dckc** [comment]
> Let's discuss in #125.

moving discussion elsewhere misses out on github's support for line-by-line comments. I recommend against that.

**Comment by dckc** [comment]
James, I suspect this PR is going to work better broken into 2 or 3 or 4 or more parts:
  - the strictly editorial tweaks at the top
    - for bonus points, move whitespace-only changes to their own commit or their own PR
  - the blockexplorer part
  - the wallet part
  - the KuCoin part

**Comment by dckc** [comment]
> We have to be very careful to make sure that everything on this page is accurate.

We have to balance that with the fact that it's already inaccurate now.

File: developer-rchain-coop/src/views/token-swap.hbs (hunk 1 above)
Comment (JamesMoc) [reply developer-rchain-coop/src/views/token-swap.hbs:3]: Leaving it in? No. I copied header info and changed it all to absolute links to test how my changes would look. Thanks.

Comment (ian-bloom) [reply developer-rchain-coop/src/views/token-swap.hbs:3]: change reverted

File: developer-rchain-coop/src/views/token-swap.hbs (hunk 2 above)
Comment (ian-bloom) [reply developer-rchain-coop/src/views/token-swap.hbs:55]: wallets listed along with disclaimer: These resources are not officially endorsed by RChain Cooperative.

File: developer-rchain-coop/src/views/token-swap.hbs (hunk 3 above)
Comment (ian-bloom) [reply developer-rchain-coop/src/views/token-swap.hbs:56]: Disclaimer should offer some protection to Co-op, but doesn't much help users.
#rhoc2rev Discord discussion channel is linked for those looking for help deciding which tools to trust.

File: developer-rchain-coop/src/views/token-swap.hbs (hunk 4 above)
Comment (ian-bloom) [reply developer-rchain-coop/src/views/token-swap.hbs:84]: agree

**Comment by ian-bloom** [comment]
@JamesMoc #128 accepted and merged to master.
I added further changes in 165fe2b055
Publication to website awaits review by @leithaus

**Pull request merged**